### ISSUE TITLE
Why does "ispaq" need both a StationXML and a RESP file?

### ISSUE THREAD

**Issue by peacocks** [EarthScope/ispaq#25]
Question:
Why does "ispaq" need both a StationXML and a RESP file?  What does one contain that the other doesn't?

I have to produce "RESP" files by running "rdseed" on a dataless SEED file output by my local data service.  Since "rdseed" is no longer supported, could "ispaq" make do with just the StationXML file?

**Comment by gillian-earthscope** [comment]
Yes, this is an issue that is on our backlog for development work for ISPAQ. We recognize that this is adversely affecting our users but we have not had the resources to address it. The part of the code that calculates frequency-amplitude-phase information used in the PSD calculation is dependent on RESP format information. All other metadata is taken from the StationXML. We do intend to fix this in the future so that StationXML is used for both.
